nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/xml/XMLRecordReader.java

```diff
@@ -84,6 +84,10 @@ public XMLRecordReader(InputStream in, RecordSchema schema, String rootName, Str
 
         try {
             final XMLInputFactory xmlInputFactory = XMLInputFactory.newInstance();
+
+            // Avoid namespace replacements
+            xmlInputFactory.setProperty(XMLInputFactory.IS_NAMESPACE_AWARE, false);

```

Comment:
Might want to avoid XEE vulnerability via improved configuration of XMLInputFactory?
https://www.owasp.org/index.php/XML_External_Entity_(XXE)_Prevention_Cheat_Sheet#XMLInputFactory_.28a_StAX_parser.29

Reply:
@tballison Thank you for the advice! I refactored this in a way that only the local part is considered. 
:)